app/lib/frontend/templates/layout.dart

```diff
@@ -57,9 +57,11 @@ String renderLayoutPage(
       ? null
       : {
           'email': userSessionData.email,
-          // Set image size to 30x30 pixels for faster loading, see:
-          // https://developers.google.com/people/image-sizing
-          'image_url': userSessionData.imageUrl + '=s30',
+          'image_url': userSessionData.imageUrl == null
+              ? null
+              // Set image size to 30x30 pixels for faster loading, see:
+              // https://developers.google.com/people/image-sizing
+              : '${userSessionData.imageUrl}=s30',
```

Comment:
This would render to: 'null=s30'.
I would prefer userSessionData.imageUrl == null ? null : '${userSessionData.imageUrl}=s30'

Reply:
There is a null check 3 lines up

Reply:
actually the code is exactly like yours :)

Reply:
Ah sorry - i was only reading this line :)